Scripts/DCS-BIOS/test/AircraftTest.lua

```diff
@@ -43,10 +43,6 @@ function TestAircraft:testFW190D9()
 	self:validateModule(require("FW-190D9"), "FW-190D9", 0x3400)
 end
 
-function TestAircraft:testBf109K4()
-	self:validateModule(require("Bf-109K-4"), "Bf-109K-4", 0x4200)
-end
-
```

Comment:
This probably shouldn't be deleted. It should maybe be moved up so it's in alphabetical order, but I don't think we want to remove these tests.

Reply:
OK, I thought it was a duplicate

Reply:
You actually might be right. The ordering is weird in the file too. We should correct it at some point so it's all in alphabetical order.